### PR TITLE
Added How it works text to button

### PR DESCRIPTION
**Pull request by kanishka8276**
Solves issue #16
Added How it works text in the link on the home page

### CODE REVIEW

File: src/components/Home/_Child247.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const Child247 = () => {
         <p>{Text}</p>
         <div className="action-button">
           <Link to={Paths.HowItWorks} className="easya-btn-nav">
-            {_.HowItWorks.Title}
+           How It works
```
Comment (praveenscience) [inline src/components/Home/_Child247.js:16]: Can we please have this translated, thanks.

**Review by praveenscience — CHANGES_REQUESTED** [review]

**Comment by praveenscience** [comment]
Any updates on this? 🤔

**Comment by kanishka8276** [comment]
> Any updates on this? 🤔

No , Not yet

**Comment by praveenscience** [comment]
@kanishka8276 I have merged it anyway... I don't want you to lose points, at least you took it up! 😊

**Comment by kanishka8276** [comment]
> @kanishka8276 I have merged it anyway... I don't want you to lose points, at least you took it up! 😊

Thanks a lot 😀

**Pull request merged**